src/ip/Intercept.cc

```diff
@@ -428,6 +417,41 @@ Ip::Intercept::Lookup(const Comm::ConnectionPointer &newConn, const Comm::Connec
     return false;
 }
 
+
+bool
+Ip::Intercept::LookupTproxy(const Comm::ConnectionPointer &newConn)
+{
+    /* --enable-linux-netfilter    */
+    /* --enable-ipfw-transparent   */
+    /* --enable-ipf-transparent    */
+    /* --enable-pf-transparent     */
+#if IPF_TRANSPARENT || LINUX_NETFILTER || IPFW_TRANSPARENT || PF_TRANSPARENT
+
+    debugs(89, 5, HERE << "address BEGIN: me/client= " << newConn->local << ", destination/me= " << newConn->remote);
+
+    /* NP: try TPROXY first, its much quieter than NAT when non-matching */
+    if (transparentActive_ && TproxyTransparent(newConn))
+        return true;
+
+#else /* none of the transparent options configured */
+    debugs(89, DBG_IMPORTANT, "WARNING: transparent proxying not supported");
+#endif
+
+    return false;
+}
+
+bool
+Ip::Intercept::SuppressErrors()
```

Comment:
**If** we change this code, then we should go two steps further:

```C++
/// XXX: TcpAcceptor::oldAccept() reports "same" errors while ignoring this code.
/// \returns debugs() level for reporting the current error
int
Ip::Intercept::errorReportingLevel()
{
    // TODO: Consider limiting level-1 errors to, say, one per minute:
    // const auto important = squid_curtime - lastReported_ > 60;
    lastReported_ = squid_curtime;
    return DBG_IMPORTANT;
}
```

The callers will need to be adjusted accordingly, of course.

AFAICT, the above sketch also fixes the `squid_curtime - lastReported_ >= 60` comment bug in the current PR code (that condition was for _reporting_ errors, not suppressing them).

All of this should really be done via cache_log_message instead of directly in each error-reporting code, but that PR has not been merged yet, so we should probably not reference that directive here for now.

----

However, if it is easier to undo all these and related changes and just pass the port information to the old Lookup(), possibly instead of one of the Connection pointers, then we probably should do that. This debugging logic is essentially just noise, but some other code modifications are difficult to get right and may encounter author resistance down the road, despite the low quality of the current official code.